src/constraints/vectorofvariables.jl

```diff
@@ -21,7 +21,7 @@ function MOI.add_constraint(model::LinQuadOptimizer, func::VecVar, set::S) where
     rows = get_number_linear_constraints(model)
     num_vars = MOI.dimension(set)
     add_linear_constraints!(model,
-        CSRMatrix{Float64}(collect(1:num_vars),
+        CSRMatrix{Float64}([collect(1:num_vars); num_vars+1],
```

Comment:
Just `collect(1:num_vars+1)`?

Reply:
This is because for a SparseMatrixCSC the column ptr list has to have a ptr to the last val+1.  This is correct.

Reply:
I meant replace `[collect(1:num_vars); num_vars+1]` with `collect(1:num_vars+1)`. Aren't they identical?